test cases/selinux/1 basic/meson.build

```diff
@@ -0,0 +1,14 @@
+project('selinux test', 'c', meson_version: '>=1.9.0')
+
+selinux = import('selinux')
+
+if not selinux.found()
+  error('SELinux development tools not found')
+endif
```

Comment:
To actually skip the tests you need to prefix the error message with `MESON_SKIP_TEST`.

Reply:
done